src/patternfly/components/FormControl/examples/form-control-input-example.hbs

```diff
@@ -1,12 +1,12 @@
-{{#> form-control controlType="input" input="true" type="text" id="textInput1" disabled="yes" aria-label="disabled text input"}}
+{{#> form-control controlType="input" form-control--attribute='disabled input="true" type="text" id="textInput1" aria-label="disabled text input"'}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" type="text" id="textInput2" aria-label="standard input example" }}
+{{#> form-control controlType="input" form-control--attribute='input="true" type="text" id="textInput2" aria-label="standard input example"'}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" type="text" id="textInput3" aria-label="readonly input example" readonly="true" value="readonly input example" readonly="true"}}
+{{#> form-control controlType="input" form-control--attribute='readonly input="true" type="text" id="textInput3" aria-label="readonly input example" value="readonly input example"'}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" type="text" id="textInput4" required="true" aria-invalid="true" aria-label="Error state input example"}}
+{{#> form-control controlType="input" form-control--attribute='required input="true" type="text" id="textInput4" aria-invalid="true" aria-label="Error state input example"'}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" pf-c-form-control--modifier="pf-m-form-control-alt" type="text" id="textInput5" aria-label="Alternative input example"}}
+{{#> form-control controlType="input" form-control--modifier="pf-m-form-control-alt" form-control--attribute='input="true" type="text" id="textInput5" aria-label="Alternative input example"'}}
```

Comment:
input="true" needs to be moved out of the attributes. It's not an attribute, It's  being used to set a ">" only if it's an input.

Reply:
removed :)